cave/com.raytheon.viz.product.awips/plugin.properties

```diff
@@ -1,6 +1,9 @@
 caveAboutText=Common AWIPS Visualization Environment (CAVE)\n\
 \n\
-Developed on the Raytheon Visualization Environment (viz)\n\
+Developed on the Raytheon viz Environment, released as non-operational\
+software by UCAR's Unidata Program Center\
+https://www.unidata.ucar.edu/software/awips \
```

Comment:
I assume this is the same as what was in v18?
Do we want to link directly to the AWIPS docs instead though?  (http://unidata.github.io/awips2/)
I sort of feel like that's a more useful page...but I'm down with whatever you think.